lib/config/presets/internal/workarounds.ts

```diff
@@ -35,6 +36,19 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  ignoreSpringCloudAlphabetic: {
+    description:
+      'Ignore spring cloud releases older than the 2020 release train',
+    packageRules: [
+      {
+        matchDatasources: ['maven'],
+        matchPackageNames: [
+          'org.springframework.cloud:spring-cloud-starter-parent',
+        ],
+        allowedVersions: '/^[0-9]/',
```

Comment:
Should we allow only ones with a more strict pattern? e.g. starting with four digits or a 2. Or does it work equally well to simply check for a leading digit?

Reply:
@tobiasgies ?

Reply:
@rarkins hey, apologies, I've been out sick for a few days now. I'll get back to this ASAP.

Reply:
@tobiasgies are you able to resume this?

Reply:
Yep, should be able to do it over the weekend. Thanks for the reminder!

Reply:
Since the 1.0.x.RELEASE versions are still there, I would personally go with `/^2/` or `/^2\d{3}/`.

I need the changes from this PR in my company, so if it keeps getting postponed, I will create another PR.

@tobiasgies Are you planning on committing something in the next few days?

Reply:
@bergerst you can always ignore this preset and add your own rule to your config. so this can't be a blocker.

Reply:
@viceice How can I exclude a workaround that's included in config:base?

Reply:
https://docs.renovatebot.com/configuration-options/#ignorepresets